README.md

```diff
@@ -20,15 +20,200 @@ tasks specified in the Gruntfile.
+8) Click on the **`Environment Variables...`** button at the bottom of the screen. The Envi-ronment Variables dialog opens.
+9) Select the **`Path`** variable in the bottom System variables scroll window.
+10) Click **`Edit...`**
+11) Click into the **`Variable value`** text box.
+12) Press your **`End`** key to move to the very end of this extremely long string.
+13) Enter ’;’ and paste the `platform-tools` directory path after that semicolon.
+14) Click on **`OK`** and exit all of the windows.
+15) Verify that you have made the change by closing all **cmd** windows and open a new one(so it picks up the change), and type
+```
+$ adb version
+```
+You should now see the version of the **adb** tool. For example: `Android Debug Bridge version 1.0.31`. You can now move on to [Installing Application Designer](https://docs.odk-x.org/app-designer-install/).
+
+###### For Mac/Unix
+The *PATH* variable is nothing more than a default list of places the system looks for commands. Open a **terminal**. Type:
+```
+$ echo $PATH
+```
+You will see a colon-separated list of folders on your computer. (echo means just print whatever comes next, and the `${ }` means that the system will treat *PATH* as a variable, not a program. You don't need to know this to follow these instructions, but knowledge is power.) For example, you might see something like this:
+```
+$ echo $PATH
+  /usr/local/bin:/usr/local/sbin:/usr/bin:/bin
+```
+This means that when you type:
+```
+$ adb --version
+```
+the system will look for the command called **adb** in the directories `/usr/local/bin/`, `/usr/local/sbin/`, `/usr/bin/`, and `/bin/`.
+Note the location where you downloaded the Android SDK. It should contain a folder called `platform-tools`, which itself contains the program **adb**. If this was in the folder `/Users/someuser/Desktop/android-sdk/` you should be able to run:
+```
+$ /Users/someuser/Desktop/android-sdk/platform-tools/adb --version
+```
+This works because we're telling the computer exactly where the program **adb** exists. By putting the `platform-tools` directory on the system's *PATH* variable, we will be able to just type **adb** and have the system find it in the `/Users/someuser/Desktop/android-sdk/platform-tools/` directory.
+This process is more involved on Mac/Unix than on Windows. Use a text editor (not **Word**, but something like **TextEdit**), select the option to open a file, and browse to your home directory. You can find your home directory by typing:
+```
+$ echo ~
+```
+in a **terminal**. ('~' is a shortcut for the home directory.) Macs use a hidden file called `.bash_profile` in the home directory to set variables like *PATH*. Other Unix systems use files like `.bashrc`. You might have to check the specifics for your distribution to know which you should use. Open the appropriate file. If the file does not already exist, create a new file that will be saved with the appropriate name in your home directory.
```

Comment:
macOS's default shell has been switched to `zsh` there isn't a `.bash_profile` anymore. See https://superuser.com/questions/866683/iterm-zsh-not-reading-bashrc-or-bash-profile for more information.